src/redux/Rocket/RocketsSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const rocketURL = 'https://api.spacexdata.com/v4/rockets';
+
+export const getRocket = createAsyncThunk('rockets/getMission', async () => {
+  const res = await fetch(rocketURL);
+  const data = await res.json();
+  return data;
+});
```

Comment:
- [ ] After fetching the rockets from the API you need to filter the data and avoid saving all of them to the store. Kindly do that to fulfill the requirements listed here: https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store

- **[OPTIONAL]** I think this should be something like `rockets/getRockets` instead of `rockets/getMission`